looker_style_guide.md

```diff
@@ -0,0 +1,184 @@
+* Measures should reference the dimension (e.g. ${order_total} over ${TABLE}.order_total)
+
+#### Example
+
+```
+view: intercom_conversations {
+  sql_table_name:
+  -- if prod -- analytics.analytics.fct_intercom_conversations
+  -- if dev -- analytics.{{_user_attributes['dbt_schema']}}.fct_intercom_conversations ;;
+
+# =============================================== DIMENSIONS
+
+# ==================== IDs
+  dimension: conversation_id {
+    group_label: "Identifiers"
+    description: "Primary key for the table. Links to the intercom conversation thread."
+    primary_key: yes
+    hidden: yes
+    type: string
+    sql: ${TABLE}."CONVERSATION_ID" ;;
+    link: {
+      label: "Intercom Link"
+      url: "https://app.intercom.com/a/apps/c15gqki8/inbox/inbox/all/conversations/{{ value }}"
+    }
+  }
+  dimension: customer_id {
+    group_label: "Identifiers"
+    description: "The identifier for the customer."
+    type: string
+    sql: ${TABLE}."CUSTOMER_ID" ;;
+  }
+
+# ==================== CONVERSATIONS
+  dimension: responses {
+    group_label: "Conversation response metrics"
+    description: "Sum of admin and user responses"
+    type: number
+    sql: ${TABLE}."TOTAL_RESPONSES" ;;
+  }
+
+# ==================== TIMESTAMPS
+  dimension_group: updated {
+    group_label: "Timestamps"
+    description: "Timestamp of last alterations EST"
+    hidden: yes
+    type: time
+    timeframes: [
+     raw,
+     time,
+     date,
+     week,
+     month,
+     quarter,
+     year
+    ]
+    sql: ${TABLE}."UPDATED_AT" ;;
+    convert_tz: no
+  }
+
+# =============================================== DRILL FIELDS
+
+  set: account_information {
+    fields: [
+      account_id,
+      account_name,
+      plan,
+      created_date
+      ]
+  }
+
+# =============================================== MEASURES
+
+# ==================== SLA PERFORMANCE   
+  measure: total_responses {
+    group_label: "SLA Performance"
+    description: "The total responses within an Intercom conversation (both admin and user)"
+    type: sum
+    sql: ${responses} ;;
+    value_format_name: percent_1
+  }
+
+  measure: total_responses_enterprise {
+    label: "Total Responses by Enterprise Customers"
+    group_label: "SLA Performance"
+    description: Total responses within an Intercom conversation for Enterprise accounts"
+    type: sum
+    sql: ${responses} ;;
+    filters: [plan: "Enterprise"]
+    drill_fields: [account_information*]
+  }
+```
+
+## Models
+* Explores should be organized by group label and group labels should be organized alphabetically
+* Every explore should be listed under a `group_label` (see how we categorize explores in the "Structure of our LookML project" section)
+* Explores should have only a few joins at max. If you're finding yourself joining several views to a single explore, it might mean you'll need to: 1. Model this in dbt 2. Rethink which table should be the base of the explore you're creating
```

Comment:
I think we should talk about joins where you limit to only include a few dimensions from a larger table. is this a pattern we want? I think yes but know we don't do it a ton in our project currently

Reply:
YES! Added this point in here